aiohttp_client_cache/backends/base.py

```diff
@@ -267,7 +267,7 @@ async def has_url(self, url: StrOrURL) -> bool:
 
     async def urls(self) -> List[str]:
```

Comment:
Can `urls` be made AsyncIterable as well? Not sure how much it's used, but in non-trivial setups this likely will hit mem limits. Not sure if the sorted here is just to make testing this simpler.

Reply:
Sure, just updated it.